app/views/users/show.html.erb

```diff
@@ -11,7 +11,7 @@
     <% @user.last_3_posts.each do |post| %>
         <%= link_to user_post_path(@user.id, post.id) do %>
         <div class="user-recent-post">
-            <p class="post-title">Post : <%= post.title %></p>
+            <p class="post-title"><strong>Post</strong> : <%= post.title %></p>
             <p class="post-text"><%= post.text %></p>
```

Comment:
- Good job so far 👍🏽  However, I would like to inform you that I could not view the users I have added to the database before running the server. I thought at first that this might be because you don't have the root route, but nothing was visible on the screen after making some changes. Kindly try adding [data](https://github.com/microverseinc/curriculum-rails/blob/main/blogapp/projects/Processing_data_model_project.md#use-models-to-insert-data) to the database to confirm what went wrong 👍🏽.

Reply:
Thank you so much for the feedback, let me work on that right away.